<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JuliusTestConfigurationWidget</class>
 <widget class="QFrame" name="JuliusTestConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string />
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tag:</string>
       </property>
       <property name="buddy">
        <cstring>leTag</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="leTag"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>HTK input files</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_8">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="lbHmmdefs">
        <property name="text">
         <string>HMM definitions:</string>
        </property>
        <property name="buddy">
         <cstring>urHmmDefs</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KUrlRequester" name="urHmmDefs"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbTiedlist_4">
        <property name="text">
         <string>Tiedlist:</string>
        </property>
        <property name="buddy">
         <cstring>urTiedlist</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbDict_4">
        <property name="text">
         <string>Dict:</string>
        </property>
        <property name="buddy">
         <cstring>urDict</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="KUrlRequester" name="urDict"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lbDFA_4">
        <property name="text">
         <string>DFA:</string>
        </property>
        <property name="buddy">
         <cstring>urDFA</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="KUrlRequester" name="urDFA"/>
      </item>
      <item row="6" column="1">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <widget class="KUrlRequester" name="urTiedlist"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string comment="Test the model">Test configuration</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_9">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbTestPrompts_3">
        <property name="text">
         <string>Test-prompts:</string>
        </property>
        <property name="buddy">
         <cstring>urTestPrompts</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KUrlRequester" name="urTestPrompts"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbTestPromptsBasePath_3">
        <property name="text">
         <string>Test-prompts base path:</string>
        </property>
        <property name="buddy">
         <cstring>urTestPromptsBasePath</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KUrlRequester" name="urTestPromptsBasePath"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbJConf_3">
        <property name="text">
         <string>JConf:</string>
        </property>
        <property name="buddy">
         <cstring>urJConf</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KUrlRequester" name="urJConf"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbSampleRate">
        <property name="text">
         <string>Samplerate:</string>
        </property>
        <property name="buddy">
         <cstring>sbSampleRate</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="sbSampleRate">
        <property name="minimum">
         <number>8000</number>
        </property>
        <property name="maximum">
         <number>128000</number>
        </property>
        <property name="value">
         <number>16000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="pbRemove">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
